src/common/options/mds.yaml.in

```diff
@@ -1577,3 +1577,13 @@ options:
   - mds
   flags:
   - runtime
+- name: mds_session_metadata_threshold
+  type: size
+  level: advanced
+  desc: Evict non-advancing client-tid sessions exceeding the config size.
+  long_desc: Evict clients which are not advancing their request tids which causes a large buildup of session metadata (`completed_requests`) in the MDS causing the MDS to go read-only since the RADOS operation exceeds the size threashold. This config is the maximum size that a session metadata (encoded) can grow.
```

Comment:
nit: the validation is the size in bytes. The description should probably be clear about that, size in bytes vs size in number of ops in the session.